bot/cogs/help_channels.py

```diff
@@ -133,6 +134,7 @@ def __init__(self, bot: Bot):
 
         # Stats
         self.claim_times = {}
+        self.unanswered = {}
```

Comment:
Can you leave a short comment explaining what's in this dict?

Reply:
Typehinted + commented both this dictionary and the dictionary above.